src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SignatureVerificationCache.cs

```diff
@@ -47,16 +47,15 @@ private ColumnMasterKeyMetadataSignatureVerificationCache()
         /// <param name="allowEnclaveComputations">boolean indicating whether the key can be sent to enclave</param>
         /// <param name="signature">Signature for the CMK metadata</param>
         /// <returns>null if the data is not found in cache otherwise returns true/false indicating signature verification success/failure</returns>
```

Comment:
Update as it no longer returns `null`

Does MemoryCache.TryGetValue return `False` if it doesn't find an item?